parsec/core/mountpoint/manager.py

```diff
@@ -19,45 +19,37 @@
     MountpointConfigurationWorkspaceFSTimestampedError,
     MountpointAlreadyMounted,
     MountpointNotMounted,
-    MountpointDisabled,
+    MountpointWinfspNotAvailable,
+    MountpointFuseNotAvailable,
 )
 from parsec.core.mountpoint.winify import winify_entry_name
 
 
 def get_mountpoint_runner():
+    # Windows
     if os.name == "nt":
-        try:
-            import winfspy  # noqa: test import is working
 
-            logging.getLogger("winfspy").setLevel(logging.WARNING)
+        try:
+            import_function("winfspy")
```

Comment:
why not using good'old `import winfspy` ?

Reply:
So I can easily mock it in the tests. Any problem with that?

Reply:
no it's fine, but we should add a comment about this then ;-(